### PR TITLE
PD-374: Add new options for Migration spec

### PR DESCRIPTION
**Pull request by andreipope**
PD-374: Add new options for Migration spec

### CODE REVIEW

File: content/portworx-install-with-kubernetes/migration/kubemotion/_index.md [hunk 1]
```diff
@@ -246,53 +246,50 @@ If you see an error similar to the one above, you should increase the size of th
...
+## Start a migration
+
+1. Create a file called `migration.yaml` file, specifying the following fields and values:
+
+  * **apiVersion:** as `stork.libopenstorage.org/v1alpha1`
+  * **kind:** as `Migration`
+  * **metadata.name:** with the name of the object that performs the migration
+  * **metadata.namespace:** with the name of the namespace in which you want to create the object
+  * **spec.clusterPair:** with the name of the `ClusterPair` object created in the [Create a ClusterPair](#create-a-clusterpair) section
+  * **spec.includeResources:** with a boolean value specifying if the migration should include PVCs and applications <!-- Are there any other resources we should mention?-->. If you set this field to `false`, Portworx will only migrate your volumes
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/migration/kubemotion/_index.md:258]: @disrani-px

Comment (disrani-px) [inline content/portworx-install-with-kubernetes/migration/kubemotion/_index.md:258]:
```suggestion
  * **spec.includeResources:** with a boolean value specifying if the migration should include PVCs and other applications specs. If you set this field to `false`, Portworx will only migrate your volumes
```

File: content/portworx-install-with-kubernetes/migration/kubemotion/_index.md [hunk 2]
```diff
@@ -246,53 +246,50 @@ If you see an error similar to the one above, you should increase the size of th
...
+  * **spec.startApplications:** with a boolean value specifying if Portworx should automatically start applications on the destination cluster. If you set this field to `false`, then the `Deployment` and `StatefulSet` objects on the destination cluster will be set to zero. Note that, on the destination cluster, Portworx uses the `stork.openstorage.org/migrationReplicas` annotation to store the number of replicas from the source cluster
+  * **spec.namespaces**: with the list of namespaces Portworx should migrate
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/migration/kubemotion/_index.md:260]:
```suggestion
  * **spec.namespaces**: with the list of namespaces to be migrated
```

File: content/portworx-install-with-kubernetes/migration/kubemotion/_index.md [hunk 3]
```diff
@@ -246,53 +246,50 @@ If you see an error similar to the one above, you should increase the size of th
...
+  * **spec.adminClusterPair:** with the name of the `ClusterPair` object created in the admin namespace that Portworx should use to migrate your cluster-scoped resources. Use this if your regular users lack permission to create or delete cluster-scoped resources on the destination cluster. If you don't specify this field, then Portworx will use the object specified in the `spec.clusterPair` field. This feature was introduced in STORK v2.3.0.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/migration/kubemotion/_index.md:261]:
```suggestion
  * **spec.adminClusterPair:** with the name of the `ClusterPair` object created in the admin namespace that Portworx should use to migrate your cluster-scoped resources. Use this if your regular users lack permission to create or delete cluster-scoped resources on the destination cluster. If you don't specify this field, then Stork will use the object specified in the `spec.clusterPair` field. This feature was introduced in STORK v2.3.0.
```

File: content/portworx-install-with-kubernetes/migration/kubemotion/_index.md [hunk 4]
```diff
@@ -246,53 +246,50 @@ If you see an error similar to the one above, you should increase the size of th
...
+  * **spec.purgeDeletedResources:** with a boolean value specifying if Portworx should automatically purge a resource from the destination cluster when you delete it from the source cluster. The default value is `false`. This feature was introduced in STORK v2.3.2.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/migration/kubemotion/_index.md:262]:
```suggestion
  * **spec.purgeDeletedResources:** with a boolean value specifying if Stork should automatically purge a resource from the destination cluster when you delete it from the source cluster. The default value is `false`. This feature was introduced in STORK v2.3.2.
```

**Pull request merged**